src/Theme.js

```diff
@@ -1,18 +1,30 @@
 import { createTheme } from "@mui/material/styles";
+import {
+  PRIMARY_COLOR_MAIN,
+  PRIMARY_COLOR_ALTERNATIVE,
+  SECONDARY_COLOR_MAIN,
+  SECONDARY_COLOR_ALTERNATIVE,
+} from "./contants/colors";
 
 // A custom theme for this app
 const theme = createTheme({
   typography: {
     fontFamily: ["Poppins", "Arial", "sans-serif"].join(","),
+    button: {
+      fontWeight: "bold",
+      textTransform: "none",
+    },
```

Comment:
Why don't you override the `fontSize` here instead of in the styled component?

Reply:
good idea.